README.md

```diff
@@ -59,8 +60,9 @@ First make sure you've got Rust **1.4.0** or greater available, then:
 
 `cargo test` to run all tests.
 
-To run rustfmt after this, use `cargo run -- filename`. See the notes above on
-running rustfmt.
+To run rustfmt after this, use `cargo run --bin rustfmt -- filename`. See the
+notes above on running rustfmt. To run cargo-fmt, use
+`cargo run --bin cargo-fmt`
```

Comment:
I will edit it and make things a bit clearer, but this section describes how to build and run rustfmt, not how to install and use it then.